Assets/_Project/Code/Scripts/Leaderboard/Leaderboardscript.cs

```diff
@@ -0,0 +1,86 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using TMPro;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
+using System.Linq;
+using System;
+
+[System.Serializable]
+public  class LeaderboardEntry:IComparable<LeaderboardEntry>
```

Comment:
Nazwa jest myląca, patrząc na to myślę że to nie dane drużyny, tylko jeden rząd zawierający obiekty tekstu, dopóki nie mamy #30, proponuje zamienić nazwę na TempTeam